### PR TITLE
Enable highlights for search term in results

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: lib/search/global/projects.js [hunk 1]
```diff
@@ -11,6 +11,15 @@ module.exports = client => async (term = '', query = {}) => {
 
   params.body.query = { bool: {} };
 
+  params.body.highlight = {
+    fields: {
+      'title': { type: 'plain', pre_tags: '**', post_tags: '**' },
+      'establishment.name': { type: 'plain', pre_tags: '**', post_tags: '**' },
+      'licenceNumber': { type: 'plain', pre_tags: '**', post_tags: '**' },
+      'licenceHolder.lastName': { type: 'plain', pre_tags: '**', post_tags: '**' }
```
Comment (lennym) [inline lib/search/global/projects.js:19]: It's probably easier to define these as:
```
fields: {
  '*': { type: 'plain', pre_tags: '**', post_tags: '**' }
}
```

Comment (wheelsandcogs) [reply lib/search/global/projects.js:19]: Wasn't aware you could do that, cheers.

Comment (lennym) [reply lib/search/global/projects.js:19]: I'm not 100% sure, but I do `content.*` in the project search, so I'm assuming a plain `*` works too.

File: lib/search/global/projects.js [hunk 2]
```diff
@@ -33,8 +39,7 @@ module.exports = client => async (term = '', query = {}) => {
   const fields = [
     'title',
     'licenceHolder.lastName',
```
Comment (lennym) [inline lib/search/global/projects.js:41]: There was chat about adding first name as well, no?

Comment (wheelsandcogs) [reply lib/search/global/projects.js:41]: Indeed, was going to fix that in a separate PR, but can do it here if you prefer?

Comment (wheelsandcogs) [reply lib/search/global/projects.js:42]: Done.

**Pull request merged**